src/libs/actions/Report.js

```diff
@@ -1152,10 +1152,11 @@ function navigateToConciergeChat() {
  * @param {Object} policy
  * @param {String} reportName
  * @param {String} visibility
+ * @param {Array} policyMembers
  */
-function addPolicyReport(policy, reportName, visibility) {
+function addPolicyReport(policy, reportName, visibility, policyMembers) {
     // The participants include the current user (admin) and the employees. Participants must not be empty.
-    const participants = _.unique([currentUserAccountID, ..._.pluck(policy.employeeList, 'accountID')]);
+    const participants = _.unique([currentUserAccountID, ...policyMembers]);
     const policyReport = ReportUtils.buildOptimisticChatReport(
```

Comment:
I can't comment on the exact line but I think since the `policy` param is now the policyID and not the policy object this is wrong https://github.com/Expensify/App/blob/d16fe7e52492fe1bd0d848886a2069fa861550ea/src/libs/actions/Report.js#L1164